### ISSUE TITLE
iOS 12 WebARonARKit

### ISSUE THREAD

**Issue by william-lawvere** [CityScope/CS_CityScopeAR#1]
Hi,

I've tried to follow your instructions and install WebARonARCore APK on iOS (https://github.com/google-ar/WebARonARKit) and then navigate to your website, but the browser keeps crashing. 

One of the reasons might be that my phone runs iOS 12, not iOS 11. The WebARonARCore package says it needs to be run on iOS 11. Have you encountered this issue in the past?

**Comment by RELNO** [comment]
Hi @william-lawvere -- WebARonARCore  was not updated for very long time (I'm afraid the project was dropped) so It wouldn't be a surprise new OSs are not supported

**Comment by william-lawvere** [comment]
Oh I see. 

Do you have any other suggestions for going about running your program? I'd like to attempt to run augmented reality simulations for my own architectural models

**Comment by RELNO** [comment]
At this point I'd suggest exploring arkit/arcore via unity. It's not as pretty as deploying to the web but looks like google/apple support for unity dev. is far more stable.

**Comment by william-lawvere** [comment]
Sounds good. Do you happen to have instructions for running your code using arkit/arcore via unity? I'm not super familiar with those frameworks, but I can pick them up

**Comment by RELNO** [comment]
Not really, we're a bit slacking on docs.. The good thing about both frameworks is that simple AR usage involves not much more than throwing a camera module into a unity scene, building and running on a device. follow this https://developers.google.com/ar/develop/unity/quickstart-android to get started.
